src/Items/ItemPotion.h

```diff
@@ -77,8 +77,7 @@ class cItemPotionHandler:
 
 		if (!a_Player->IsGameModeCreative())
 		{
-			a_Player->GetInventory().RemoveOneEquippedItem();
-			a_Player->GetInventory().AddItem(cItem(E_ITEM_GLASS_BOTTLE));
+			a_Player->GetInventory().SetEquippedItem(cItem(E_ITEM_GLASS_BOTTLE));
```

Comment:
The changes use Replace...TossRest elsewhere, should glass bottles use it too?

Reply:
That was an oversight.